diff --git a/api/bazel/repository_locations.bzl b/api/bazel/repository_locations.bzl
index c66c73a..54793de 100644
--- a/api/bazel/repository_locations.bzl
+++ b/api/bazel/repository_locations.bzl
@@ -22,7 +22,6 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         use_category = ["api"],
         implied_untracked_deps = [
             "com_github_iancoleman_strcase",
-            "com_github_lyft_protoc_gen_star",
             "com_github_spf13_afero",
             "org_golang_google_genproto",
             "org_golang_x_text",
